packages/components/src/ListView/Items/Items.scss

```
@@ -16,6 +16,13 @@ $row-nested-inner-margin-bottom: $padding-larger - $padding-small;
 		:global {
 			.checkbox {
 				margin-left: $padding-smaller;
+				display: flex;
+			}
+			:global(.tc-svg-icon) {
```

you already are in a `:global`
```suggestion
			.tc-svg-icon {
```